deploy/deployConfig_test.go

```diff
@@ -32,9 +32,12 @@ someOtherValue = "foo"
 [[deployment.targets]]
 Name = "name1"
 URL = "url1"
+CloudFrontDistributionID = "cdn1"
```

Comment:
I'm not sure we ever will get support for something other than CF, but I don't think you will have more than one at a time -- so we might as well call this `CDNDistributionID` or something. We can use CloudFront in help texts etc., but at least make room for another should it happen.

Reply:
If we did add a different provider, we'd need a way to identify which provider. So, I could change this to `CDNDistributionID`, but then if we add another one later we'd still need to add a new field like `CDNProvider`, with values like `cloudfront` and `fooprovider`.

I considered this but decided to leave it more explicit; if we add another provider we would add it as `FooProviderDistributionID`.

Let me know which way you want to go here.